syntax = "proto3";

package import_package;

message response_msg {
}
message request_msg {
}
